DeveloperGame/DeveloperGame/ComputerGeekGame.cs

```diff
@@ -148,5 +200,97 @@ private void PrintLeaderBoard()
                 Console.WriteLine($"Name: {playerScore.Name} | Point(s): {playerScore.Points} | Time: {playerScore.Time}\n");
             }
         }
+
+        private void RandomizeMultipleChoiceQuestions()
+        {
+            List<MultipleChoiceQuestion> tempQuestionList = new List<MultipleChoiceQuestion>();
+            Random random = new Random();
+
+            tempQuestionList.AddRange(_multipleChoiceQuestions);
+            _multipleChoiceQuestions.Clear();
+
+            while (tempQuestionList.Count > 0)
+            {
+                int randomIndex = random.Next(0, tempQuestionList.Count - 1);
+
+                _multipleChoiceQuestions.Add(tempQuestionList[randomIndex]);
+                tempQuestionList.Remove(tempQuestionList[randomIndex]);
+            }            
+        }
+
+        private void RandomizeTrueOrFalseQuestions()
+        {
+            List<TrueOrFalseQuestion> tempQuestionList = new List<TrueOrFalseQuestion>();
+            Random random = new Random();
+
+            tempQuestionList.AddRange(_trueOrFalseQuestions);
+            _trueOrFalseQuestions.Clear();
+
+            while (tempQuestionList.Count > 0)
+            {
+                int randomIndex = random.Next(0, tempQuestionList.Count - 1);
+
+                _trueOrFalseQuestions.Add(tempQuestionList[randomIndex]);
+                tempQuestionList.Remove(tempQuestionList[randomIndex]);
+            }            
+        }
+
+        private MultipleChoiceQuestion.Answers GetMcAnswerFromPlayer()
+        {               
+
+            while (true)
+            {
+                Console.Write("\n\tAnswer: ");
+                string answerStg = Console.ReadLine();
+
+                if(answerStg.Length < 1 || string.IsNullOrWhiteSpace(answerStg))
+                {
+                    Console.WriteLine("Please make sure you enter a valid answer.");
+                    Console.Write("\n\tAnswer: ");
+                    answerStg = Console.ReadLine();
+                }
+
+                char answerChar = char.ToUpper(answerStg[0]);
+
+                if (answerChar == 'A')
+                    return MultipleChoiceQuestion.Answers.A;
+
+                else if (answerChar == 'B')
+                    return MultipleChoiceQuestion.Answers.B;
+
+                else if (answerChar == 'C')
+                    return MultipleChoiceQuestion.Answers.C;
+
+                else if (answerChar == 'D')
+                    return MultipleChoiceQuestion.Answers.D;                
+            }
+
+            
+        }
+
+        private bool GetTfAnswerFromUser()
+        {          
+            while (true) 
+            {
+                Console.Write("\n\tAnswer: ");
+                string answerStg = Console.ReadLine();
+
+
+                if (answerStg.Length < 1 || string.IsNullOrWhiteSpace(answerStg))
```

Comment:
Whitespace, also same issue with the loop, it seems like your tried to do 2 different ways at the same time. You tell them invalid answer every other time